### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by mesbahamin** [thebearjew/thebearjew.github.io#25]
The following link is broken:
http://jry.io/posts/make-your-jekyll-blog-awesome-with-git-submodules/

**Comment by thebearjew** [comment]
Hey Amin!
Thanks for pointing this out. I'm currently taking down my blog for several reasons, but if you'd like to still read the post you can find it here:

https://github.com/thebearjew/blog-posts/blob/master/2015-11-16-make-your-jekyll-blog-awesome-with-git-submodules.md

Funny that when I finally decide to remove the blog, someone is interested in it haha!

**Comment by mesbahamin** [comment]
Yeah I was just thinking about how to separate my blog posts from the code for my website, and your post came up in the Google results. I'm glad you still had the markdown file in a public repo, and I found your post helpful :)

I ended up just using a symlink instead of a submodule. I still find submodules a bit... unintuitive.

**Comment by thebearjew** [comment]
@mesbahamin Great! Glad to here it. They are slightly unintuitive, they might make more sense when you're working on a large codebase and manage your own internal packages + dependencies.
It's pure magic then
